### ISSUE TITLE
kvas dns test изменяет dns сервер по умолчанию на 9.9.9.9 когда по умолчанию установлен AGH

### ISSUE THREAD

**Issue by ergashev** [qzeleza/kvas#27]
Приветствую!
Хотел посмотреть правильные ли ip мне возвращает АGH, но наткнулся на ошибку из-за которой изменяется dns сервер по умолчанию(AGH) на 9.9.9.9.
На сколько я смог понять логику скрипта, нужно поправить два момента:
1. [В этой строке](https://github.com/qzeleza/kvas/blob/main/opt/bin/libs/vpn#L1611)  заменить `&> /dev/null` на `2> /dev/null` для того что бы переменная ips не была всегда пустой. 
2. Из-за пустой ips, строкой ниже срабатывает условие в котором DNS_DEFAULT перезаписывается значением из DNS_STATIC, в котором и содержится 9.9.9.9

**Comment by wildrun0** [comment]
Такое происходит и без AGH насколько я заметил, тоже

**Comment by qzeleza** [comment]
> 1. [В этой строке](https://github.com/qzeleza/kvas/blob/main/opt/bin/libs/vpn#L1611)  заменить `&> /dev/null` на `2> /dev/null` для того что бы переменная ips не была всегда пустой.

Доброго дня,
Вы у себя пробывали исправлять? Поделитесь результатом?

**Comment by ergashev** [comment]
> Доброго дня, Вы у себя пробывали исправлять? Поделитесь результатом?
Да, для меня это работает. Но я изменил только первый пункт, т.к. мне этого оказалось этого достаточно.
Сейчас, когда у меня установлен AGH в качестве DNS по умолчанию, то `kvas dns test` и` kvas dns test example.com` отрабатывают через AGH. Но если вдруг DNS AGH будет недоступен, то "DNS сервер по умолчанию" изменится на 9.9.9.9

**Comment by qzeleza** [comment]
Вышла [бета 22](https://github.com/qzeleza/kvas/blob/main/ipk/), пробуем. 
Жду обратной связи по данному этикету.

**Comment by qzeleza** [comment]
Прошу дать обратную связь поданному этикету, в противном случае, тикет будет закрыт через 3 дня.

**Comment by ergashev** [comment]
Добрый день! Проблема решена в 22 бете. Тикет можно закрыть. Но в 22 бете обнаружил что 
перестала корректно работать проверка статуса VPN интерфейса(определяется как отключённый интерфейс, хотя он включен). А также появилась проверка статуса интернет соединения, которая тоже не работает для меня корректно (у меня PPPoE), квас при тесте все время предлагает перезапустить роутер, хотя соединение есть.
К сожалению, пока не обладаю достаточным количеством времени для того что бы помочь подробнее разобраться в сути проблемы и выгрузить дебаг логи или поизучать изменения в коде. Т.к. не было времени оценить влияние новых проблем, я временно откатился на 21 бету. Если на неделе с этой проблемой никто с похожими проблемами не обратится, то я заведу новый тикет как появится время. Пока могу лишь пару скриншотов скинуть настройки 22 беты.
![IMG_20221211_150020](https://user-images.githubusercontent.com/20318751/206898797-1082cbe7-6e57-4e69-ae16-5e1cd99433f0.jpg)
![IMG_20221211_150507](https://user-images.githubusercontent.com/20318751/206898812-da98da34-9b74-4950-a944-f204887b8140.jpg)
![IMG_20221211_150811](https://user-images.githubusercontent.com/20318751/206898851-009be3a1-3252-439d-b132-9d38af5f41eb.jpg)
![IMG_20221211_151530](https://user-images.githubusercontent.com/20318751/206898860-85582e26-5858-4857-8257-eba16b6d7887.jpg)
